### ISSUE TITLE
diff and detect report different similarity scores?

### ISSUE THREAD

**Issue by tieguy** [benbalter/licensee#347]
### Describe the bug

`licensee diff` and `licensee detect` report slightly different similarity scores on https://github.com/ThreeTen/threetenbp/raw/master/LICENSE.txt

(Version: licensee 9.10.1)

### Steps to reproduce the behavior

1. run `licensee detect` on threetenbp/. result:

`  Closest non-matching licenses:`
`    BSD-3-Clause-Clear similarity:  91.34%`

2. run `licensee diff --license bsd-3-clause-clear` on LICENSE.txt in threetenbp:

` Comparing to BSD 3-Clause Clear License:`
` Input Length:      1402`
` License length:    1556`
` Similarity:      94.65%`

### Expected behavior

Similarity scores should be identical, but are not.

### Additional context

Note that BSD-3-Clause is actually a closer match than BSD-3-Clause-Clear, filed as #348. Also, possibly related to #344?

**Comment by benbalter** [comment]
On `master` I get the following results:

```
be licensee detect --remote https://github.com/ThreeTen/threetenbp
License:        NOASSERTION
Matched files:  LICENSE.txt
LICENSE.txt:
  Content hash:  10024377604b3673aaa87223ee03fd38c7205e6b
  License:       NOASSERTION
  Closest non-matching licenses:
    BSD-3-Clause-Clear similarity:  92.13%
```

```
be licensee diff --license bsd-3-clause-clear --remote https://github.com/ThreeTen/threetenbp
Comparing to BSD 3-Clause Clear License:
Input Length:      1557
License length:    1562
Similarity:      92.13%
diff --git a/LICENSE b/LICENSE
index eed934d..1ff0601 100644
--- a/LICENSE
+++ b/LICENSE
@@ -1,21 +1,20 @@
{+/* - copyright (c) 2007-present, stephen colebourne and michael nascimento+}
{+santos - * all rights reserved. - *+} redistribution and use in source and binary
forms, with or without {+-+} modification, are permitted[-(subject to the limitations in the disclaimer below)-] provided that the following
conditions are met: - redistributions of source code must retain the above
copyright notice, {+-+} this list of conditions and the following disclaimer. -
redistributions in binary form must reproduce the above copyright notice, {+-+} this
list of conditions and the following disclaimer in the documentation {+-+} and/or
other materials provided with the distribution. - neither the name of [-the copyright holder-]{+jsr-310+}
nor the names of its contributors {+-+} may be used to endorse or promote products
derived from this software {+-+} without specific prior written permission. [-no express or implied licenses to any party's patent rights-]
[-are granted by this license.-]{+- *+} this
software is provided by the copyright holders and contributors {+-+} "as is" and any
express or implied warranties, including, but not {+-+} limited to, the implied
warranties of merchantability and fitness for {+-+} a particular purpose are
disclaimed. in no event shall the copyright holder or {+-+} contributors be liable
for any direct, indirect, incidental, special, {+-+} exemplary, or consequential
damages (including, but not limited to, {+-+} procurement of substitute goods or
services; loss of use, data, or {+-+} profits; or business interruption) however
caused and on any theory of {+-+} liability, whether in contract, strict liability,
or tort (including {+-+} negligence or otherwise) arising in any way out of the use
of this {+-+} software, even if advised of the possibility of such damage. {+*/+}
```

```
be licensee diff --license bsd-3-clause --remote https://github.com/ThreeTen/threetenbp
Comparing to BSD 3-Clause "New" or "Revised" License:
Input Length:      1557
License length:    1420
Similarity:      94.69%
diff --git a/LICENSE b/LICENSE
index e57f70e..1ff0601 100644
--- a/LICENSE
+++ b/LICENSE
@@ -1,19 +1,20 @@
{+/* - copyright (c) 2007-present, stephen colebourne and michael nascimento+}
{+santos - * all rights reserved. - *+} redistribution and use in source and binary
forms, with or without {+-+} modification, are permitted provided that the following
conditions are met: - redistributions of source code must retain the above
copyright notice, {+-+} this list of conditions and the following disclaimer. -
redistributions in binary form must reproduce the above copyright notice, {+-+} this
list of conditions and the following disclaimer in the documentation {+-+} and/or
other materials provided with the distribution. - neither the name of [-the copyright holder-]{+jsr-310+}
nor the names of its contributors {+-+} may be used to endorse or promote products
derived from this software {+-+} without specific prior written permission. {+- *+} this
software is provided by the copyright holders and contributors {+-+} "as is" and any
express or implied warranties, including, but not {+-+} limited to, the implied
warranties of merchantability and fitness for {+-+} a particular purpose are
disclaimed. in no event shall the copyright holder or {+-+} contributors be liable
for any direct, indirect, incidental, special, {+-+} exemplary, or consequential
damages (including, but not limited to, {+-+} procurement of substitute goods or
services; loss of use, data, or {+-+} profits; or business interruption) however
caused and on any theory of {+-+} liability, whether in contract, strict liability,
or tort (including {+-+} negligence or otherwise) arising in any way out of the use
of this {+-+} software, even if advised of the possibility of such damage. {+*/+}
```

It's not properly normalizing the border, which is a bug in itself, but I believe the similarity percentages are as expected?

**Comment by tieguy** [comment]
FWIW, in the released version I'm testing with (9.10.1), it does normalize the border correctly, so looks like a regression? Can't test master locally today but will try to get to it tomorrow.

(q: what are you using to do the actual diff? If you're shelling out to a system diff, possibly part of the problem here is that I'm testing this on a Mac rather than a Linux box?)

**Comment by tieguy** [comment]
I can't duplicate in master either so closing.
